pkg/component/web.go

```diff
@@ -72,7 +72,7 @@ func (c *Component) initWeb() error {
 	}
 
 	if c.config.HTTP.PProf.Enable {
-		g := web.RootRouter().NewRoute().Subrouter()
+		g := web.Router().NewRoute().Subrouter()
```

Comment:
We should keep using the root router here, otherwise this will break in Enterprise. Same below.